demo/MMSegmentation_Tutorial.ipynb

```diff
@@ -49,9 +49,13 @@
    "outputs": [],
    "source": [
     "# Check nvcc version\n",
+    "import torch\n",
+    "import torchvision\n",
```

Comment:
I think not everyone has torch installed, so it would be better to install torch before call `import torch`